defaults/network-ents.go

```diff
@@ -11,10 +12,17 @@ const (
 )
 
 const (
-	NetworkNoneEntId  = "none"  // network.none
-	NetworkUserEntId  = "user"  // network.user
-	NetworkProxyEntId = "proxy" // network.proxy
-	NetworkAdminEntId = "admin" // network.admin
+	NetworkNoneEntFullId  = NetworkTLD + ".none"
```

Comment:
Hmm not sure I understand why you want this *instead*? CLI tools fronting the container management platforms would have access to these strings in order to make the parsing of the entitlements provided by the user. That way they can just check those strings in the option list and do an `entitlements.Manager.AddDefault()` of these.

Agree for the `TLD`, `networkDomain` makes sense. Will fix!